### DISCUSSION TITLE
Adding hitsounds to argon skin doesn't apply them

### DISCUSSION THREAD

**Discussion by SolsticeSpectrum** [ppy/osu#28070] [Q&A] [answered]
As mentioned here https://github.com/ppy/osu/discussions/24239
I have put Gameplay/Argon folder to the exported argon skin which I exported by pressing `Skin layout editor`, then exitting that and exporting the copy.
I then put the folder in and packed it back to osk but the hitsounds are not applying for some reason

Do they need to be specific sample rate or am I doing something wrong?
I tried `Argon`, `Gameplay/Argon` and `Samples/Gameplay/Argon`

Here is the skin
[- avenir sounds - (rabidpan).osk.zip](https://github.com/ppy/osu/files/15190338/-.avenir.sounds.-.rabidpan.osk.zip)

Also while we're at it, is it possible to skin `Textures/Gameplay/osu`? It contains the textures for some of the elements

**Comment by peppy** [comment, marked as answer]
Please use the classic skin for skinning in such a way. Replacing argon textures/samples is not supported yet and when it is it will be done via the editor.
